includes/admin/reporting/reports.php

```diff
@@ -2560,11 +2560,17 @@ function display_export_report() {
                         <div class="inside">
                             <p><?php esc_html_e( 'Download a CSV giving a detailed look into earnings over time.', 'easy-digital-downloads' ); ?></p>
                             <form id="edd-export-earnings" class="edd-export-form edd-import-export-form" method="post">
-								<?php echo EDD()->html->month_dropdown( 'start_month' ); ?>
-								<?php echo EDD()->html->year_dropdown( 'start_year' ); ?>
-								<span class="edd-label"><?php echo _x( 'to', 'Date one to date two', 'easy-digital-downloads' ); ?></span>
-								<?php echo EDD()->html->month_dropdown( 'end_month' ); ?>
-								<?php echo EDD()->html->year_dropdown( 'end_year' ); ?>
+								<div class="to-and-from-container">
```

Comment:
I think wrapping the grouped selects together here is a great idea.

Looking at the HTML, I'm suddenly wondering about 1) should this use the `.edd-form-group` markup, although that may be out of context of this PR; and 2) I'm not sure that any of these inputs have labels (also probably out of context, although would be great to fix while you're fiddling with the HTML anyway?).

Reply:
Issue [#8112](https://github.com/easydigitaldownloads/easy-digital-downloads/issues/8112)